index.html

```diff
@@ -9,25 +9,73 @@
   </head>
   <body>
     <header class="header">
-      <a href="#" class="logo-link">
+      <nav class="navbar">
+        <a href="#" class="logo-link"
+          >My logo
+          <h3 class="my-logo"></h3>
+        </a>
+        <ul class="right-nav">
+          <li class="menu-li">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="menu-li">
+            <a href="#about" class="nav-link">About</a>
+          </li>
+          <li class="menu-li">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+
+    <!-- </div> -->
+
+    <!-- <img src="./images/rightunion.png" class="hamb" alt="normal button" />
+        </div> -->
+
+    <!-- <div class="menu">
+        <img src="./images/XIcon.png" class="xicon" alt="crossicon" />
+          <ul>
+            <li class="menu-li">
+              <a href="#portfolio" class="nav-link">Portfolio</a>
+            </li>
+            <li class="menu-li">
+              <a href="#about" class="nav-link">About</a>
+            </li>
+            <li class="menu-li">
+              <a href="#contact" class="nav-link">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav> -->
+
+    <!-- <div>
+            <button id="btn" onclick="myFunction()"><img src="./images/rightunion.png" alt="menuicons"></button>
+      </div> -->
+    <!-- <a href="#" class="logo-link">
         <div>
           <h3 class="my-logo">My logo</h3>
         </div>
       </a>
-      
+
       <nav>
         <div class="right-nav">
           <img src="./images/rightunion.png" alt="normal button" />
         </div>
       </nav>
-      <nav class="right-links">
+      <nav class="right-links right-links1">
         <ul>
-          <li><a href="#"></a>Portfolio</li>
-          <li><a href="#"></a>About</li>
-          <li><a href="#"></a>Contact</li>
+          <li><a href="#portfolio"></a>Portfolio</li>
+          <li><a href="#about"></a>About</li>
+          <li><a href="#contact"></a>Contact</li>
         </ul>
       </nav>
-    </header>
+    </header> -->
```

Comment:
- [ ] Your mobile menu still doesn't stick to the Figma design. This can come from your CSS which seems to not be implemented as well. It would be better if you take some time to fix it properly as you don't have a partner.

**This is how it looks:**
![image](https://user-images.githubusercontent.com/80612925/174877676-3bbfc4f9-30b3-4417-98ea-b8830b052f0d.png)